src/main/java/mate/academy/rickandmorty/Application.java

```diff
@@ -1,12 +1,27 @@
 package mate.academy.rickandmorty;
 
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class Application {
+    private static CharacterService characterService;
+
+    @Autowired
+    public Application(CharacterService characterService) {
+        Application.characterService = characterService;
+    }
 
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return args -> characterService.saveAllToDB();
+    }
```

Comment:
You can directly inject to @Bean the needed bean.
```suggestion
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

    @Bean
    public CommandLineRunner commandLineRunner(CharacterService characterService) {
        return args -> characterService.saveAllToDB();
    }
```